pkg/descheduler/controllers/migration/arbitrator/filter.go

```diff
@@ -464,3 +476,72 @@ func (r *Reconciler) filterLimitedObject(pod *corev1.Pod) bool {
 	}
 	return true
 }
+
+func (f *filter) initObjectLimiters() {
+	var trackExpiration time.Duration
+	for _, v := range f.args.ObjectLimiters {
+		if v.Duration.Duration > trackExpiration {
+			trackExpiration = v.Duration.Duration
+		}
+	}
+	if trackExpiration > 0 {
+		f.objectLimiters = make(map[types.UID]*rate.Limiter)
+		limiterExpiration := trackExpiration + trackExpiration/2
+		f.limiterCache = gocache.New(limiterExpiration, limiterExpiration)
+		f.limiterCache.OnEvicted(func(s string, _ interface{}) {
+			f.limiterLock.Lock()
+			defer f.limiterLock.Unlock()
+			delete(f.objectLimiters, types.UID(s))
+		})
+	}
+}
+
+func (f *filter) checkExpectedPhaseAndAnnotation(a *PhaseAndAnnotation, b *PhaseAndAnnotation, uid types.UID) bool {
+	if f.checkJobPassedArbitration(uid) {
+		a.annotations[AnnotationPassedArbitration] = "true"
+	}
+	return a.phase == b.phase && isContain(a.annotations, b.annotations)
```

Comment:
There is a conflict scene here. Once the user reconfigures these limiters, for example, the maximum number of evictions per node is changed from 3 to 2, and there are currently three pmjs that have passed arbitration, then once the new configuration takes effect, it is possible that these three pmjs and subsequent new pmjs will all Cannot be executed because AnnotationPassedArbitration is still concerned.

Reply:
Yes, I will use local map to record all the `PassedArbitration` PMJs.